src/main/java/me/itzg/helpers/modrinth/ModrinthCommand.java

```diff
@@ -236,8 +247,14 @@ private Stream<? extends Path> processProject(String projectRef) {
 
         final Project project = getProject(projectRef);
         if (projectsProcessed.add(project.getId())) {
-            final List<Version> versions = getVersionsForProject(project.getId());
-            final Version version = pickVersion(versions);
+            final Version version;
+            if (projectRef.contains(":")) {
```

Comment:
and here conditional on parts.length == 2

Reply:
Should I handle the length > 3 as well or just fallback to the default method?

Reply:
The cool thing is if you use [this one](https://docs.oracle.com/en/java/javase/11/docs/api/java.base/java/lang/String.html#split(java.lang.String,int)) you can limit the split to no more than 2 elements.